.autover/changes/1c879063-ec4b-4330-80a2-3f2116812150.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "AWS.Deploy.CLI",
+      "Type": "Patch",
```

Comment:
should this be minor version since we are changing the behavior slightly?

Reply:
we are fixing the behavior so this should be fine to do a patch. however, since we mostly always do Minors for this package, i'll change to minor.